CBinding/Project/CProject.cs

```diff
@@ -229,6 +253,10 @@ public void WriteMDPkgPackage (ConfigurationSelector configuration)
 			string pkgfile = Path.Combine (BaseDirectory, Name + ".md.pc");
 			
 			CProjectConfiguration config = (CProjectConfiguration)GetConfiguration (configuration);
+			while (config == null) {
+				Thread.Sleep (20);
```

Comment:
This looks like it's hack to fix a race. It should at the very least have a FIXME comment. Fixing the race for real would be much better.